WrathCombo/Combos/PvP/BRDPVP.cs

```diff
@@ -41,47 +54,54 @@ internal class BRDPvP_BurstMode : CustomCombo
             protected internal override CustomComboPreset Preset { get; } = CustomComboPreset.BRDPvP_BurstMode;
 
             protected override uint Invoke(uint actionID, uint lastComboMove, float comboTime, byte level)
+
             {
 
                 if (actionID == PowerfulShot)
                 {
                     var canWeave = CanWeave(actionID, 0.5);
+                    uint harmonicCharges = GetRemainingCharges(HarmonicArrow);
 
                     if (!PvPCommon.IsImmuneToDamage())
                     {
+                        if (IsEnabled(CustomComboPreset.BRDPvP_Wardens) && InPvP() &&  //Autowardens set up only for soft ccs, it cant be used while cced like purify
```

Comment:
I could be mistaken, but I think this `InPVP()` check is completely unnecessary inside of PVP combos.
If it's not, that should probably be changed at some point 😅

Reply:
I would have to check. I belive i was assuming that inpvp check ensues you are actually in pvp combat actively?

Reply:
It does not check that, only if you're in a pvp zone.

Reply:
The check is not done. It is not done because the combo doesn't even run unless a PVP-only ability is being used, `Powerful Shot`.
So this check doesn't provide anything.
Not to say that it is a real problem either, still an approval from me, but it'd be best removed.